pixart.js

```diff
@@ -1 +1,14 @@
+$("#set-color").on("click", function() {
+  event.preventDefault();
+  var color = $("#color-field").val();
+  $(".brush").css("background-color", color);
+});
+var setBrushColor = $(".button").on("click", setBrushColor);
 
+for (var i = 0; i < 8000; i++) {
+  $("body").append($("div class=square></div"));
+}
+
+$("square").on("mouseover", (function(){
+  $(this).css("background-color", color).prop("value");
```

Comment:
This works, but the `.prop("value")` part at the end is probably unnecessary... curious what led you to add it?